functionary/core/models/team.py

```diff
@@ -17,3 +18,18 @@ class Team(models.Model):
 
     def __str__(self):
         return self.name
+
+    def _post_save(self, creating):
+        """Post save hooks"""
+        if creating:
+            Environment = apps.get_model("core", "Environment")
+            Environment.create_default(self)
```

Comment:
Does this save as well?

Reply:
Yes, the save is included.  Code for that is [here](https://github.com/beer-garden/functionary/pull/19/files#diff-ed90f8d5958e918da90367e7d5e9f95188811a78ea0dfe66499f1b6c3bb711d4R56).